src/chrome/content/rules/Booklooker.de.xml

```diff
@@ -1,9 +1,20 @@
+<!--
+	Non-functional hosts
+		Different content:
+		- bjoerk.booklooker.de
+-->
 <ruleset name="booklooker.de">
-  <target host="booklooker.de" />
-  <target host="*.booklooker.de" />
+	<target host="booklooker.de" />
+	<target host="www.booklooker.de" />
+	<target host="api.booklooker.de" />
+	<target host="bms.booklooker.de" />
+	<target host="dev.booklooker.de" />
+	<target host="download.booklooker.de" />
+	<target host="falco.booklooker.de" />
+	<target host="images.booklooker.de" />
+	<target host="images2.booklooker.de" />
```

Comment:
Same as `falco` and `dev`. I'm open to discussion but I'm inclined to remove it.